src/main/kotlin/eu/europa/ec/eudi/sdjwt/SdJwtVerifier.kt

```diff
@@ -273,13 +273,13 @@ fun interface UnverifiedIssuanceFrom<out JWT> {
 typealias JwtAndClaims = Pair<Jwt, JsonObject>
 
 /**
- * A single point for verifying SD-JWTs in both [Combined Issuance Format][verifyIssuance]
- * and [Combined Presentation Format][verifyPresentation]
+ * A single point for verifying SD-JWTs in both SD-JWT and SD-JWT+KB formats, using either compact or
+ * JWS JSON serialization.
  */
 interface SdJwtVerifier<JWT> {
 
     /**
-     * Verifies an SD-JWT (in simple format)
+     * Verifies an SD-JWT serialized using compact serialization.
      * Typically, this is useful to Holder that wants to verify an issued SD-JWT
```

Comment:
Consider the following wording:

 Typically, this is useful to Holder that wants to verify an issued SD-JWT or to Verifier
 that wants to verify a presented SD-JWT in case the KB-JWT [must not be present]KeyBindingVerifier.MustNotBePresent]